ts/parts/Axis.ts

```diff
@@ -2906,21 +2903,6 @@ extend(Axis.prototype, /** @lends Highcharts.Axis.prototype */{
      */
     defaultYAxisOptions: {
 
-        /**
-         * In a polar chart, this is the angle of the Y axis in degrees, where
-         * 0 is up and 90 is right. The angle determines the position of the
-         * axis line and the labels, though the coordinate system is unaffected.
-         *
-         * @sample {highcharts} highcharts/yaxis/angle/
-         *         Dual axis polar chart
-         *
-         * @type      {number}
-         * @default   0
-         * @since     4.2.7
-         * @product   highcharts
-         * @apioption yAxis.angle
-         */
-
```

Comment:
Why this is removed?

Reply:
Right now the angle property can be used both on, xAxis and yAxis, so I have moved it to the RadialAxis.ts part instead.

Reply:
Got it!